src/sage/libs/ntl/ntl_GF2X_linkage.pxi

```diff
@@ -339,12 +339,23 @@ cdef inline int celement_pow(GF2X_c* res, GF2X_c* x, long e, GF2X_c *modulus, lo
         x^9 + x^8 + x^7 + x^5 + x^3
         sage: pow(f, 2, h)
         x^9 + x^8 + x^7 + x^5 + x^3
+        sage: pow(x, 1000, h)
+        x^8 + x^7 + x^4
+
+    Check that deg x >= deg modulus works (:trac:`35324`)::
```

Comment:
Could you replace `:trac:` with `:issue:`? It is the new format for automatically linking to the doc.

Reply:
Done